### ISSUE TITLE
Accessibility - buttons should have discernible text

### ISSUE THREAD

**Issue by jaimeiniesta** [papercups-io/papercups#885]
**Describe the bug**
The button to toggle the chat widget doesn't have discernible text.

**To Reproduce**
1. Go to a page with the Papercups chat widget installed.
2. Check accessibility with [axe DevTools](https://chrome.google.com/webstore/detail/axe-devtools-web-accessib/lhdoppojpmngadmnindnejefpokejbdd)
3. See `Buttons must have discernible text` issue on `<button class="Papercups-toggleButton css-1lzvboz">`

**Expected behavior**
All buttons should have discernible text, so that screen readers can properly present the page to people with low or no vision. 

**Additional context**
Fixing this is basically following one of the solutions in https://dequeuniversity.com/rules/axe/4.2/button-name?application=rocketvalidator - basically adding a `aria-label` attribute with the text `Open chat widget` for example.

**Comment by reichert621** [comment]
thanks for reporting this @jaimeiniesta!

(the component for reference: https://github.com/papercups-io/chat-widget/blob/9d398470c0903602257dbcd37db12c582ddf0587/src/components/WidgetToggle.tsx#L153)

**Comment by jyggiz** [comment]
Hey, could i take this issue?

**Comment by reichert621** [comment]
sure @jyggiz! let me know if you need any guidance :)

**Comment by jyggiz** [comment]
@reichert621 Hello, how are you? Does platform supports support multiple languages or is it planned to do so? I have an option where, inside ChatWidget, I can hardcode the aria-label value, or pass it as a prop for multilingual purposes. What do you think, at this stage it is better to use the first option? Also thought based on props isOpen to pass open/close text :)

**Comment by reichert621** [comment]
@jyggiz feel free to hardcode it for now, we plan to support i18n eventually but it will be easy to switch to a dynamic prop once we decide to do so :)

**Comment by jyggiz** [comment]
@reichert621 Here is PR - https://github.com/papercups-io/chat-widget/pull/97 !

**Comment by devtayls** [comment]
@reichert621 Can we close this issue?
